src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.List;
+
+@Entity
+@Table(name = "shoppingCarts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne
+    @MapsId
+    private User user;
+
+    public Long getId() {
+        return id;
+    }
+
+    public ShoppingCart setId(Long id) {
```

Comment:
AGAIN, setters shouldn't return values.
```suggestion
    public void setId(Long id) {
```